docs/source/examples/Notebook/Working With Markdown Cells.ipynb

```diff
@@ -189,6 +190,7 @@
    "source": [
     "The Notebook webapp supports Github flavored markdown meaning that you can use triple backticks for code blocks:\n",
     "\n",
+    "Make sure change the cell to markdown from Cell menu. \n",
```

Comment:
If this needs clarification, it should probably be at the start of the notebook, because the whole document is talking about markdown cells, not just this bit.